source/security/private-endpoints.txt

```diff
@@ -252,3 +325,49 @@ You can toggle this setting in three ways:
 - In the Admin API, call the :admin-api-endpoint:`Toggle Non-VPC Requests
   <operation/allNonVpcClientRequests>` with a boolean value (``true`` to remove
   the default restriction, or ``false`` to reinstate it.)
+
+Definitions
```

Comment:
I figured it would be good to have a reference of sorts that explains the differences between these things. I haven't quite gotten to writing those definitions. If you have suggestions, I'd love to hear them!